sass/grid/columns.sass

```diff
@@ -243,9 +243,13 @@
     align-items: center
   // Responsiveness
   +tablet
-    &:not(.is-desktop)
+    &:not(.is-desktop):not(.is-widescreen)
```

Comment:
if i make it `is-tablet` and remove all `:not()` columns will not be activated by default from tablet onwards.